experiment/compatibility-versions/kind-upgrade.sh

```diff
@@ -112,7 +114,7 @@ usage()
 # Main
 build_docker
 IMAGES_PATH="${KUBE_ROOT}/_output/release-images/amd64"
```

Comment:
let's parameterize amd64, some of the CI machines are arm64, we can use `$(go env GOARCH)`

Reply:
This is not necessary after Michael's refactor. `IMAGES_PATH="${TMP_DIR}/kubernetes/server/bin"`